### ISSUE TITLE
iRCD does not switch between Construction and Airlocks categories

### ISSUE THREAD

**Issue by GreytideSkye** [Skyrat-SS13/Skyrat-TG#24421]
Reporting client version: 514.1589

## Round ID:
9685

## Testmerges:
- [URL Clickabillity. OOC Notes/Flavor Text/Species lore](https://github.com/Skyrat-SS13/Skyrat-tg/pull/24068)
- [Disables pipe gas visuals for now](https://github.com/Skyrat-SS13/Skyrat-tg/pull/24264)
- [[READY] [MODULAR] Blueshift wakes up after taking "Worst Nap Ever" [MDB IGNORE]](https://github.com/Skyrat-SS13/Skyrat-tg/pull/23604)

## Reproduction:

Pull out RCD. Observe the category is Construction. If you have added Upgrades to it, observe you can freely switch between the Structures, Machines, and Furniture tabs. If you haven't, observe there is only the Structures tab. Observe you can select build anything in that tab as normal.

Click the Airlocks category. 
Expected: it changes to Airlocks, shows airlock tabs and whatnot.
Observed: The Airlocks category button turns green, and the Construction category turns not-green, as if you'd changed. The list of tabs does NOT change, still having Structures (Machines and Furniture). Attempt to click on a different buildable. It does not change the currently selected buildable. Attempt to click on a different tab. You can change tabs to the other buildables, but clicking those buildables ALSO does not change the currently selected one.

Click Airlock Access category. It switches the pane to the usual access changing one.
Click Airlocks category. It pulls up the Constructions category stuff, with the Airlocks button green. Leave it on Airlocks.

Close the RCD pop-up entirely.

Open the RCD pop-up, it's the Airlocks category, and the pane below is the Airlocks tabs and the airlocks buildables.
Click the Construction category, it makes the construction category green but it's still the Airlocks tabs.

basically clicking the construction or airlock category does not refresh the pane below to actually SHOW the stuff from that category, until you close and open it again

**Comment by Iajret** [comment]
seems to be an upstream issue since even regular rcds cant switch to airlocks tab

**Comment by Iajret** [comment]
https://github.com/tgstation/tgstation/pull/79021

**Comment by vinylspiders** [comment]
Fixed https://github.com/Skyrat-SS13/Skyrat-tg/pull/24466
